builder/exoscale/config.go

```diff
@@ -90,6 +91,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		return nil, errs
 	}
 
+	if config.APITimeout == 0 {
+		config.APITimeout = 600
```

Comment:
In my opinion, this value should match the one from the `post-processor`.
What takes (too) long is registering template and it's the associated comment.

```go
// Template registration can take a _long time_, raising
// the Exoscale API client timeout as a precaution.
egoscale.ClientOptWithTimeout(time.Hour),
```

Reply:
Thanks a lot!